### ISSUE TITLE
Method call on field access

### ISSUE THREAD

**Issue by chanattan** [flix/flix#8162]
This is the original function from the file `main/src/library/Debug.flix` working with the old import system:

```scala
@Internal
    pub def debugWithPrefix(prefix: String, x: a): a \ IO =
        import java.io.PrintStream.println(String): Unit \ IO;
        println(System.out, "${prefix}%{x}");
        x
```

This is the expected refactoring:

```scala
@Internal
    pub def debugWithPrefix(prefix: String, x: a): a \ IO =
        System.out.println("${prefix}%{x}");
        x
```

Which causes the following error, probably related to parsing:

```scala
  -- Resolution Error -------------------------------------------------- Debug.flix
  
  >> Undefined name 'System.out.println'.
  
  14 |         System.out.println("${prefix}%{x}");
               ^^^^^^^^^^^^^^^^^^
               name not found
```

Parentheses fix the problem:

```scala
@Internal
    pub def debugWithPrefix(prefix: String, x: a): a \ IO =
        (System.out).println("${prefix}%{x}");
        x
```

**Comment by magnus-madsen** [comment]
The challenge is that: `System.out.println` should be understood as a GetStaticField followed by InvokeMethod2.

Not sure whether to fix this in the Parser, Weeder, or Resolver.

**Comment by JonathanStarup** [comment]
Fixed by #8193
